src/backend/core/admin.py

```diff
@@ -110,3 +110,29 @@ class TeamAdmin(admin.ModelAdmin):
         "updated_at",
     )
     search_fields = ("name",)
+
+
+@admin.register(models.Invitation)
+class InvitationAdmin(admin.ModelAdmin):
+    """Admin interface to handle invitations."""
+
+    fields = (
+        "email",
+        "team",
+        "role",
+        "created_at",
```

Comment:
nit: missing issuer ?

Reply:
Was added in the next PR, had forgotten to bring it back here. It's dooone !